torch/_jit_internal.py

```diff
@@ -744,6 +744,8 @@ def __getitem__(self, types):
 
 # Retrieves a fully-qualified name (module hierarchy + classname) for a given obj.
 def _qualified_name(obj):
+    if hasattr(obj, '_jit_override_qualname'):
+        return getattr(obj, '_jit_override_qualname')
```

Comment:
lint looks legit

also: can we add a comment describing what this is for for future people wondering?